### PR TITLE
Replace 'colorful' with 'colorama'

### PR DESCRIPTION
**Pull request by marcelometal**
We don't need two or more color packages, so I'm replacing `colorful` with `colorama`. 
I'm choosing `colorama` because there is a Debian package for it.

### CODE REVIEW

**Review by scorphus — APPROVED** [review]
So far so good 👍

**Comment by scorphus** [comment]
May I suggest rebasing on top of current master and see if the checks become green?

**Comment by marcelometal** [comment]
> May I suggest rebasing on top of current master and see if the checks become green?

Nops...  But I will fix it soon

```
thumbor/doctor.py:420:8: W0[6](https://github.com/thumbor/thumbor/runs/5778889615?check_suite_focus=true#step:11:6)03: Using the global statement (global-statement)
thumbor/doctor.py:420:8: C0103: Constant name "Fore" doesn't conform to UPPER_CASE naming style (invalid-name)
```

File: thumbor/doctor.py [hunk 1]
```diff
@@ -405,9 +425,10 @@ def load_config(config_path):
 
 
 def configure_colors(nocolor):
-    cf.use_style("solarized")
+    init(autoreset=True)
```
Comment (scorphus) [inline thumbor/doctor.py:428]:
```suggestion
```

File: thumbor/doctor.py [hunk 2]
```diff
@@ -405,9 +425,10 @@ def load_config(config_path):
...
     if nocolor:
-        cf.disable()
+        global SHOULD_USE_COLOR  # pylint: disable=global-statement
+        SHOULD_USE_COLOR = False
```
Comment (scorphus) [inline thumbor/doctor.py:431]:
```suggestion
        init(strip=True, convert=False)
        init(strip=True, convert=False)
```

File: thumbor/doctor.py [hunk 3]
```diff
@@ -29,6 +29,8 @@
 ERROR = "⛔"
 QUESTION = "❓"
 
+SHOULD_USE_COLOR = True
+
```
Comment (scorphus) [inline thumbor/doctor.py:33]:
```suggestion
```

File: thumbor/doctor.py [hunk 4]
```diff
@@ -58,15 +60,35 @@ def get_options():
     }
 
 
-def header(msg, color=cf.yellow):
-    print(color(msg))
+def print_message(color, msg):
+    print(f"{color}{msg}" if SHOULD_USE_COLOR else msg)
```
Comment (scorphus) [inline thumbor/doctor.py:64]:
```suggestion
    print(f"{color}{msg}")
```

File: thumbor/doctor.py (hunk 2 above)
Comment (scorphus) [inline thumbor/doctor.py:431]:
```suggestion
        init(strip=True, convert=False)
    else:
        init(autoreset=True)
```

**Review by scorphus — COMMENTED** [review]
Let me suggest another approach for turning colors off. What do you think, @marcelometal?

**Comment by scorphus** [comment]
Here's the code snippet that helped me play with `colorama`'s `init`:

```python
from colorama import Back, Fore, Style, init

# init(autoreset=True)
init(strip=True, convert=False)

print(Fore.RED + 'some red text')
print(Back.GREEN + 'and with a green background')
print(Style.DIM + 'and in dim text')
print(Style.RESET_ALL)
print('back to normal now')
```

**Comment by marcelometal** [comment]
> Let me suggest another approach for turning colors off. What do you think, @marcelometal?

Nice! code updated, thanks!

File: thumbor/doctor.py [hunk 5]
```diff
@@ -58,15 +58,35 @@ def get_options():
     }
 
 
-def header(msg, color=cf.yellow):
-    print(color(msg))
+def print_message(color, msg):
+    print(f"{color}{msg}")
```
Comment (scorphus) [inline thumbor/doctor.py:62]: Now this is a nit: perhaps now we can get rid of this method. That means one less function call. One less function in the code... Would you see any benefit in maintaining it?

**Review by scorphus — APPROVED** [review]

File: thumbor/doctor.py (hunk 5 above)
Comment (marcelometal) [reply thumbor/doctor.py]: done!

**Review by scorphus — APPROVED** [review]
🤘 :shipit:

**Pull request merged**